### DISCUSSION TITLE
Temp sensor not working

### DISCUSSION THREAD

**Discussion by lordratner** [PricelessToolkit/Droplet#6] [Q&A] [answered]
I just realized there's a temp sensor on the main board, but for some reason it's not giving a value in HA. It just shows "unknown."

![Screenshot_20230122-214124](https://user-images.githubusercontent.com/49489398/213962143-a8e0d4a4-cb60-4f3d-84d5-960324383931.png)

I do not have the expansion board. Is this normal?

**Comment by PricelessToolkit** [comment, marked as answer]
https://esphome.io/components/sensor/dallas.html

**Comment by lordratner** [comment]
Ok got it! Might be useful to have in the setup instructions for any ESPHome newbies like me. 

Is that the only sensor that needs to be set up without the expansion board?

**Comment by PricelessToolkit** [comment]
today I will write instruction, yes only temp sensor
